zio-http-testkit/src/main/scala/zio/http/TestServer.scala

```diff
@@ -109,7 +109,7 @@ final case class TestServer(driver: Driver, bindPort: Int) extends Server {
         ),
       )
 
-  override def port: Int = bindPort
+  override def port[R]: URIO[R, Int] = ZIO.succeed(bindPort)
```

Comment:
Delete the `R` type parameter.

Reply:
Done. Not sure why I would have added that in the first place :thinking: